native/dll/Win32DLL/SageTVWin32DLL.cpp

```diff
@@ -709,7 +703,9 @@ JNIEXPORT jlong JNICALL Java_sage_Sage_getEventTime0(JNIEnv *env, jclass jc)
 //    }
 	// NARFLEX - I have no idea why this was using GetMessageTime; but that causes the value to
 	// vary between threads which is very, very bad so we don't use that at all anymore.
+#pragma warning(disable:28159) // Static Code Analysis: When we stop supporting XP we can use GetTickCount64()
```

Comment:
Can't you just put this once at the top of the file? Putting it 4 times in the same function looks really ugly.

Reply:
You're right it is ugly. I was trying not to globally disable warnings, but this one really could/should be disabled at the top of the file.
Done